tests/test_apis/test_basic_int.py

```diff
@@ -9,7 +8,7 @@ def test_int_output():
     lng = -122.418307270836
 
     assert h3.latlng_to_cell(lat, lng, 9) == 617700169958293503
-    assert h3.latlng_to_cell(lat, lng, 9) == 0x8928308280fffff
```

Comment:
Can we change this rule? We use lower case 'f' pretty much everywhere else.

Reply:
I don't think we can.

The linter has a bunch of configurations, but the fixer—which is the thing that actually makes these changes—is much more opinionated. We _could_ just skip the fixer, but then the dev workflow would involve running the linter and manually addressing any issues that arise ... 

Given this specific occurrence just happens once in a test, we could also just mare this one line to be skipped?

Reply:
Sure. Let's just skip it for now, and maybe complain upstream to see if we can get a fixer option for this :)